tables/tRO/recvpackets.txt

```diff
@@ -1,1582 +1,1584 @@
+0ABA 2 2 0
+0ABB 2 2 0
+0ABC -1 4 0
+0ABD 10 10 0
+0ABE -1 6 0
+0ABF -1 4 0
+0AC0 26 26 0
+0AC1 26 26 0
+0AC2 -1 5 0
+0AC3 2 2 0
+0AC4 -1 64 0
+0AC5 156 156 0
+0AC6 156 156 0
+0AC7 156 156 1
+0AC8 2 2 0
+0AC9 -1 47 0
+0ACA 3 3 0
+0ACB 12 12 0
+0ACC 18 18 0
+0ACD 23 23 0
+0ACE 4 4 0
+0ACF 68 68 0
+0AD0 11 11 0
+0AD1 -1 8 0
+0AD2 30 30 1
+0AD3 -1 4 1
+0AD4 -1 4 1
+0AD5 2 2 0
+0AD6 2 2 1
+0AD7 8 8 0
+0AD8 8 8 0
+0AD9 -1 6 1
+0ADA 32 32 1
+0ADB -1 8 0
+0ADC 6 6 1
+0ADD 24 24 1
+0ADE 6 6 0
+0ADF 58 58 0
+0AE0 30 30 0
+0AE1 28 28 0
+0AE2 7 7 1
+0AE3 -1 28 0
+0AE4 89 89 1
+0AE5 -1 28 1
+0AE6 10 10 0
+0AE7 38 38 0
+0AE8 2 2 0
+0AE9 13 13 0
+0AEC 2 2 0
+0AED 2 2 0
+0AEE 2 2 0
+0AEF 2 2 0
+0AF0 10 10 0
+0AF1 102 102 0
+0AF2 40 40 1
+0AF3 -1 4 1
+0AF4 11 11 0
+0AF5 3 3 0
+0AF6 88 88 0
+0AF7 32 32 0
+0AF8 11 11 0
+0AF9 6 6 0
+0AFA 58 58 0
+0AFB -1 4 1
+0AFC 16 16 0
+0AFD -1 8 1
+0AFE -1 6 1
+0AFF -1 8 1
+0B00 8 8 1
+0B01 56 56 0
+0B02 26 26 0
+0B03 -1 47 0
+0B04 190 190 0
+0B05 63 63 1
+0B07 -1 47 0
+0B08 -1 5 1
+0B09 -1 5 1
+0B0A -1 5 1
+0B0B 4 4 1
+0B0C 155 155 1
+0B0D 10 10 1
+0B0E -1 4 1
+0B0F -1 4 1
+0B10 10 10 1
+0B11 4 4 1
+0B12 2 2 -1
+0B13 48 48 0
+0B14 2 2 0
+0B15 7 7 0
+0B16 2 2 0
+0B17 3 3 0
+0B18 4 4 0
+0B19 2 2 0
+0B1A 29 29 1
+0B1B 2 2 1
+0B1C 2 2 0
+0B1D 2 2 0
+0B1E 14 14 0
+0B1F 14 14 1
+0B20 271 271 1
+0B21 13 13 0
+0B22 5 5 0
+0B23 6 6 0
+0B24 6 6 1
+0B25 6 6 1
+0B27 -1 4 1
+0B28 3 3 1
+0B2B 11 11 1
+0B2C 3 3 1
+0B2D 11 11 1
+0B2E 4 4 1
+0B2F 73 73 1
+0B30 -1 4 1
+0B31 17 17 1
+0B32 -1 4 1
+0B33 17 17 1
+0B34 50 50 0
+0B35 3 3 0
+0B36 -1 14 1
+0B37 -1 47 1
+0B39 -1 5 1
+0B3C 4 4 1
+0B3D -1 12 1
+0B3E -1 8 1
+0B3F 64 64 1
+0B40 -1 8 1
+0B41 70 70 1
+0B42 62 62 1
+0B43 48 48 1
+0B44 58 58 1
+0B45 58 58 1
+0B46 10 10 0
+0B47 14 14 1
+0B48 18 18 0
+0B49 4 4 0
+0B4A 6 6 0
+0B4B 4 4 0
+0B4C 2 2 0
+0B4D -1 6 0
+0B4E -1 6 1
+0B4F 2 2 0
+0B50 2 2 0
+0B51 2 2 0
+0B52 2 2 0
+0B53 52 52 0
+0B54 8 8 0
+0B55 -1 4 0
+0B56 -1 8 1
+0B57 -1 4 1
+0B58 2 2 1
+0B59 4 4 1
+0B5A -1 34 1
+0B5B 14 14 1
+0B5C 2 2 1
+0B5D 10 10 1
+0B5E 33 33 1
+0B5F -1 5 0
+0B60 -1 64 0
+0B61 -1 47 0
+0B62 -1 17 1
+0B63 -1 24 1
+0B64 -1 7 1
+0B65 -1 4 1
+0B66 26 26 1
+0B67 33 33 1
+0B68 12 12 1
+0B69 18 18 1
+0B6A -1 63 0
+0B6B 14 14 1
+0B6C 12 12 1
+0B6D 6 6 0
+0B6E 14 14 0
+0B6F 177 177 0
+0B70 -1 8 0
+0B71 177 177 0
+0B72 -1 4 0
+0B73 8 8 1
+0B74 1026 1026 0
+0B75 1026 1026 0
+0B76 77 77 1
+0B77 -1 4 1
+0B78 -1 4 1
+0B79 -1 8 1
+0B7A -1 4 1
+0B7B 118 118 1
+0B7C -1 4 1
+0B7D -1 4 1
+0B7E 60 60 1
+0B7F 10 10 1
+0B80 10 10 0
+0B8C -1 5 1
+0B8D -1 5 1
+0B8E 18 18 1
+0B8F 6 6 0
+0B90 2 2 0
+0B91 8 8 0
+0B92 5 5 0
+0B93 12 12 0
+0B94 14 14 0
+0B95 -1 4 0
+0B96 26 26 0
+0B97 27 27 0
+0B98 6 6 0
+0B99 10 10 0
+0B9A 11 11 0
+0B9B 12 12 0
+0B9C 16 16 0
+0B9D 14 14 0
+0B9E 12 12 0
+0B9F 10 10 0
+0BA0 2 2 0
+0BA1 3 3 0
+0BA2 10 10 0
+0BA3 10 10 0
+0BA4 85 85 1
+0BA5 12 12 1
+0BA6 -1 8 1
+0BA7 -1 8 1
+0BA8 7 7 1
+0BA9 -1 8 1
+0BAA 22 22 1
+0BAB 22 22 1
+0BAC 22 22 0
+8B00000A -1 4 1
```

Comment:
to merge just need to fix this packet

Reply:
@sctnightcore

Reply:
done: 38cf8593230a9e504c296ad2cd13ff1d56c8e630